### ISSUE TITLE
inconsistent date localization in polls-list

### ISSUE THREAD

**Issue by tzwenn** [fsr-itse/1327#673]
Upon completing a poll I receive a banner with an English-l10n'd date.
Might just be my browser (`Accept-Language: en-us`), yet I should be consistent with other dates on the same page.
![screen shot polls](https://user-images.githubusercontent.com/495690/49144283-4b5f8900-f2fd-11e8-8306-5b833ad4b0aa.png)

**Comment by He3lixxx** [comment]
Happens to me too, Firefox 63.0.3, preferred language set to English.

**Comment by janno42** [comment]
true, this is one of multiple places with missing localization, which are unfortunately not yet listed anywhere besides in my private notes. thanks for opening the issue!
